src/app/auth/+state/auth.service.ts

```diff
@@ -13,4 +13,27 @@ export class AuthService extends FireAuthService<AuthState> {
     super(store);
   }
 
+  onCreate() {
```

Comment:
Why did you created that ? This will flood the console of all project using it.

Reply:
it is in the example app. To test the hooks, shouldn't flood all the project users?

Reply:
ah sorry I didn't realized it was the example app...